### ISSUE TITLE
jspm install adds node_modules to package override ignore

### ISSUE THREAD

**Issue by teohhanhui** [jspm/jspm-cli#1214]
Steps to reproduce:

```shell
jspm install --lock --latest npm:moment@^2.10.6 -o '{"directories":{"lib":"src"},"main":"moment.js","ignore":["./src/test/"]}'
jspm install --lock --latest npm:moment-msdate@^0.1.0 npm:moment-range@^2.0.3
```

You will see under `jspm.overrides`:

```json
"npm:moment@2.10.6": {
  "directories": {
    "lib": "src"
  },
  "main": "moment.js",
  "ignore": [
    "./src/test/",
    "node_modules",
    "node_modules"
  ]
}
```

**Comment by lookfirst** [comment]
WAT? Screenshot: http://take.ms/M9Q8d

**Comment by teohhanhui** [comment]
@lookfirst `jspm.overrides` in `package.json`

**Comment by teohhanhui** [comment]
And I'm not sure how your comment is relevant to this issue...

**Comment by guybedford** [comment]
@teohhanhui this issue will no longer happen in jspm 0.17 so I'm closing it as a wont fix for 0.16.

@lookfirst when I install moment from npm, I only get `process` as a dependency.
